src/input-validator.ts

```diff
@@ -69,3 +69,20 @@ export function checkEmulatorBuild(emulatorBuild: string): void {
 function isValidBoolean(value: string): boolean {
   return value === 'true' || value === 'false';
 }
+
+export function checkDiskSize(diskSize: string): void {
```

Comment:
would be good to add some tests for this, but don't stress about it😃

Reply:
Implemented tests uncovered a corner case, which is already handled :)